system/wine/wine.SlackBuild

```diff
@@ -78,6 +79,15 @@ case "$ARCH" in
   ;;
 esac
 
+MULTILIB=${MULTILIB@L}
+if [[ "$ARCH" != "x86_64" && "$MULTILIB" = "yes" ]]; then
```

Comment:
I don't think these conditionals are correct. There is now no way to build on x86_64 but only WINE64.